libs-sol/nearbridge/contracts/NearBridge.sol

```diff
@@ -31,11 +31,13 @@ contract NearBridge is INearBridge {
         mapping(uint256 => BlockProducer) next_bps;
     }
 
-    uint256 constant public LOCK_ETH_AMOUNT = 1 ether;
-//    uint256 constant public LOCK_DURATION = 1 hours;
-    uint256 constant public LOCK_DURATION = 10 seconds;
+    uint256 constant public DEFAULT_LOCK_ETH_AMOUNT = 1 ether;
+//    uint256 constant public DEFAULT_LOCK_DURATION = 1 hours;
+    uint256 constant public DEFAULT_LOCK_DURATION = 10 seconds;
 
     bool public initialized;
+    uint256 public lock_eth_amount;
+    uint256 public lock_duration;
```

Comment:
Is it in seconds?

Reply:
Yes, also common practice to be in second: https://solidity.readthedocs.io/en/v0.4.24/units-and-global-variables.html#time-units